GitUI/GitUICommands.cs

```diff
@@ -512,7 +512,7 @@ public bool DoActionOnRepo(Func<bool> action)
 
         #region Checkout
 
-        public bool StartCheckoutBranch(IWin32Window owner, string branch, bool remote, string[] containRevisons)
+        public bool StartCheckoutBranch(IWin32Window owner, string branch = "", bool remote = false, string[] containRevisons = null)
```

Comment:
:memo: It was previously impossible to call `StartCheckoutBranch` and specify `branch` without specifying `remote`. This increases the number of calling combinations allowed for the method and requires each of the combinations be carefully considered.

Reply:
True, though the overload `StartCheckoutBranch(string branch, bool remote)` is never called anywhere anyway 😆

If this is merged I'll make another PR that goes through and removes unused overloads. Or I can add that to this PR if preferred.

Reply:
> Or I can add that to this PR if preferred.

No!

Reply:
🤣